src/lib.rs

```diff
@@ -3,4 +3,4 @@
 #![allow(non_snake_case)]
 #![allow(improper_ctypes)]
 
-include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+include!("bindings.rs");
```

Comment:
This should be reverted - the bindings should not be committed to the repository they should be generated at compile-time.